cosmos/profiles/bigquery/oauth.py

```diff
@@ -0,0 +1,36 @@
+"Maps Airflow GCP connections to dbt BigQuery profiles that uses oauth via gcloud, if they don't use key file or JSON."
+from typing import Any
```

Comment:
```suggestion
"Maps Airflow GCP connections to dbt BigQuery profiles that uses oauth via gcloud, if they don't use key file or JSON."
from __future__ import annotations

from typing import Any
```

Reply:
I think adding `from __future__ import annotations` should fix this

Reply:
Thanks for the suggestion. I have added this 😄